client/authorizer/authorizer.go

```diff
@@ -32,7 +32,12 @@ func New(ctx context.Context, opts ...client.ConnectionOption) (*Client, error)
 	}, err
 }
 
+// Close closes the underlying connection.
+func (c *Client) Close() error {
+	return c.conn.Close()
+}
+
 // Connection returns the underlying grpc connection.
-func (c *Client) Connection() grpc.ClientConnInterface {
+func (c *Client) Connection() *grpc.ClientConn {
```

Comment:
@gertd This is the one part I'm not sure about. I added a `Close()` function on `Client` so perhaps it would be better to keep the return value of `Connection()` as `grpc.ClientConnInterface` so that users can't close the connection underneath the client's feet.

Reply:
Maybe naive, but I was expecting we return a `*grpc.ClientConn` where we return a `grpc.ClientConnInterface` right now, so we do not have to add a Close() method, as it is already present on the `*grpc.ClientConn`

Reply:
I revised the PR to eliminate `client.Connection` entirely. 
`client.NewConnection` now returns the `*grpc.ClientConn` with no additional wrapping.